python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -233,8 +234,9 @@ def since_last_handled(self: "AutomationCondition[AssetKey]") -> "SinceCondition
         with disable_dagster_warnings():
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +232]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->update doc block?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24700?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->